Lombiq.Tests.UI/Services/WebDriverFactory.cs

```diff
@@ -20,76 +20,73 @@ public static class WebDriverFactory
 
     public static ChromeDriver CreateChromeDriver(BrowserConfiguration configuration, TimeSpan pageLoadTimeout)
     {
-        var config = new ChromeConfiguration { Options = new ChromeOptions().SetCommonOptions(), Service = null };
-
-        ChromeDriver CreateDriverInner()
+        ChromeDriver CreateDriverInner(ChromeDriverService service)
         {
+            var config = new ChromeConfiguration { Options = new ChromeOptions().SetCommonOptions() };
+
             config.Options.AddArgument("--lang=" + configuration.AcceptLanguage);
 
             config.Options.SetLoggingPreference(LogType.Browser, LogLevel.Info);
 
-            // Disabling the Chrome sandbox can speed things up a bit, so recommended when you get a lot of timeouts
-            // during parallel execution:
+            // Disabling the Chrome sandbox can speed things up a bit, so it's recommended when you get a lot of
+            // timeouts during parallel execution:
             // https://stackoverflow.com/questions/22322596/selenium-error-the-http-request-to-the-remote-webdriver-timed-out-after-60-sec
             // However, this makes the executing machine vulnerable to browser-based attacks so it should only be used
             // with trusted code (like our own).
             config.Options.AddArgument("no-sandbox");
 
             // Linux-specific setting, may be necessary for running in containers, see
-            // https://developers.google.com/web/tools/puppeteer/troubleshooting#tips
+            // https://developers.google.com/web/tools/puppeteer/troubleshooting#tips for more information.
             config.Options.AddArgument("disable-dev-shm-usage");
 
             if (configuration.Headless) config.Options.AddArgument("headless");
```

Comment:
Originally I didn't dare to name it `config` because there is already a configuration and assumed that would be confusing.
Now it's going to be your fault. 😈

Reply:
Renaming them right now!

Reply:
Done.

Reply:
Named it `chromeConfig`.

Reply:
I still feel a bit guilty of using an abbreviation 🤔 now that you've brought up the topic.